packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -146,6 +146,27 @@ export function PortableTextInput(props: PortableTextInputProps) {
     }
   }, [hasFocusWithin])
 
+  const setFocusPathFromEditorSelection = useCallback(
+    (focusPath: Path) => {
+      // Report focus on spans with `.text` appended to the reported focusPath.
+      // This is done to support the Presentation tool which uses this kind of paths to refer to texts.
+      // The PT-input already supports these paths the other way around.
+      // It's a bit ugly right here, but it's a rather simple way to support the Presentation tool without
+      // having to change the PTE's internals.
+      if (
+        focusPath.length === 3 &&
```

Comment:
Paraphrasing tommy, why is the` length === 3` here? What does it means when it's `3`?

Reply:
It means that the `focusPath` doesn't point to inside an inline object, but terminates at the editor level.